src/views/TableDetail.vue

```diff
@@ -141,6 +164,18 @@ export default Vue.extend({
       return result;
     },
   },
+  watch: {
+    workspace() {
+      this.update();
+    },
+    table() {
+      this.update();
+    },
+
+    pagination() {
+      this.update();
+    },
```

Comment:
We can remove this whitespace

Reply:
I want to include whitespace like this between method declarations for readability. I made the change here in the interest of starting a new habit. But as future work occurs I plan to introduce this type of whitespace in the other modules.

My thinking is just to have surrounding blank lines for multi-line functions in the component declarations. If you have a series of simple one-liners, you could choose to group those in a block, but in general I'd say the extra whitespace drastically improves legibility.

Reply:
It just wasn't consistent with the other functions here. I agree with your comment on readability though